### PR TITLE
feat: add SAS_EXPRESS_KEEP_ALIVE_TIMEOUT

### PR DESCRIPTION
**Pull request by TarikGul**
closes: https://github.com/paritytech/substrate-api-sidecar/issues/1231

Allow sidecar to expose `SAS_EXPRESS_KEEP_ALIVE_TIMEOUT` as a enviornment variable.  It expects a number value, and it defaults to 5 is if nothing inputted (That is the default express value).

```
SAS_EXPRESS_KEEP_ALIVE_TIMEOUT: number;
```

### CODE REVIEW

**Review by marshacb — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -157,8 +157,7 @@ For more information on our configuration manager visit its readme [here](https:
 
 - `SAS_EXPRESS_BIND_HOST`: address on which the server will be listening, defaults to `127.0.0.1`.
 - `SAS_EXPRESS_PORT`: port on which the server will be listening, defaults to `8080`.
-- `SAS_EXPRESS_LOG_MODE`: enable console logging of "all" HTTP requests, only "errors", or nothing by
-    setting it to anything else. LOG_MODE defaults to only "errors".
+- `SAS_EXPERSS_KEEP_ALIVE_TIMEOUT`: Set the `keepAliveTimeout` in express.
```
Comment (TarikGul) [inline README.md:160]: Fix naming here

File: src/main.ts [hunk 2]
```diff
@@ -89,7 +89,10 @@ async function main() {
 	});
 
 	// Start the server
-	app.listen();
+	const server = app.listen();
+
+	server.keepAliveTimeout = config.EXPRESS.KEEP_ALIVE_TIMEOUT;
+	server.headersTimeout = config.EXPRESS.KEEP_ALIVE_TIMEOUT + 5;
```
Comment (tetz-akaneya) [inline src/main.ts:95]: 5 should be 5000

Comment (TarikGul) [reply src/main.ts:95]: Totally correct thanks!

File: README.md [hunk 3]
```diff
@@ -157,8 +157,7 @@ For more information on our configuration manager visit its readme [here](https:
 
 - `SAS_EXPRESS_BIND_HOST`: address on which the server will be listening, defaults to `127.0.0.1`.
 - `SAS_EXPRESS_PORT`: port on which the server will be listening, defaults to `8080`.
-- `SAS_EXPRESS_LOG_MODE`: enable console logging of "all" HTTP requests, only "errors", or nothing by
```
Comment (CurlyBracketEffect) [inline README.md:160]: Is `SAS_EXPRESS_LOG_MODE` being removed as a valid configuration value?

**Review by CurlyBracketEffect — APPROVED** [review]
The functionality looks good to me. I just have the one question about why the `SAS_EXPRESS_LOG_MODE` line is being removed from the Readme

File: README.md (hunk 3 above)
Comment (TarikGul) [reply README.md:160]: Funny, but also weirdly enough it's a very old legacy mode which has been removed for a while now.

**Pull request merged**